### PR TITLE
Add word_count uBenchmark

### PR DESCRIPTION
**Pull request by boramalper**
Signed-off-by: Bora M. Alper <bora@boramalper.org>

### CODE REVIEW

File: function-images/tests/word_count/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM golang:1.16.6-buster AS builder
```
Comment (ustiugov) [inline function-images/tests/word_count/Dockerfile:1]: use the vhiveease go image instead

File: function-images/tests/word_count/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+WORKDIR /word_count
+COPY . .
+RUN make
+
+FROM debian:buster
```
Comment (ustiugov) [inline function-images/tests/word_count/Dockerfile:6]: use scratch (at least when debugging is done)

File: function-images/tests/word_count/word-count.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        - image: docker.io/vhiveease/word_count:volatile
+          imagePullPolicy: Always
+          args: ["--verbose", "--out", "s3://vhive-corral-bench/word_count/"]
+          env:
+            - name: KNATIVE
+              value: "1"
+            - name: AWS_REGION
+              value: us-west-1
+          ports:
+            - name: h2c
```
Comment (ustiugov) [inline function-images/tests/word_count/word-count.yml:41]: this works for HTTP/2 (gRPC) but may not work for the original HTTP

**Review by ustiugov — CHANGES_REQUESTED** [review]

**Comment by boramalper** [comment]
@ustiugov Ready

File: function-images/tests/word_count/word_count.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+		count++
+	}
+	emitter.Emit(key, strconv.Itoa(count))
+}
+
+func main() {
+	job := corral.NewJob(wordCount{}, wordCount{})
+
+	options := []corral.Option{
+		corral.WithSplitSize(10 * 1024),
```
Comment (ustiugov) [inline function-images/tests/word_count/word_count.go:41]: How can one configure those? Only in the code?

**Review by ustiugov — APPROVED** [review]

**Pull request merged**